packages/blazor-workspace/NimbleBlazor/package.json

```diff
@@ -26,6 +26,6 @@
   ],
```

Comment:
@rajsite have any visions of what we should do with this PR? Looks like the error that's currently reported is coming from eslint 9.0.0. I believe javascript-styleguide isn't ready to uptake that any time soon so one option would be to pin it to 8.x and see if the PR can complete the other major upgrades. I don't mind taking a stab at that if it sounds like the right direction.

Reply:
Don't think we need to pin it. The semver `^` skips major versions, the rennovate config is to try out with new major versions ignoring that. Think we just need to opt-out eslint:

https://github.com/ni/nimble/blob/7401855164df710e15a3d9e1bd5182a1a8e1b745/.github/renovate.json#L28

We also need to opt out Arrow from here and nuget. That one is pinned and needs to stay in sync between npm and nuget so should probably be manually handled.

Reply:
#2321